src/TTextEdit.cpp

```diff
@@ -501,21 +506,44 @@ int TTextEdit::drawGrapheme(QPainter& painter, const QPoint& cursor, const QStri
         // character.
         // mk_wcwidth_cjk does the same except it returns 2 instead of 1 for
         // characters that have an "ambiguous" East Asian width:
-        switch (mWideAmbigousWidthGlyphs ? mk_wcwidth_cjk(unicode) : mk_wcwidth(unicode)) {
+        // replaced by wide_wcwidth which returns 1 or 2 or a number of
+        // (negative) special values:
+        // mWideAmbigousWidthGlyphs
```

Comment:
Ooops - a left over place where I parked a copy of the variable whilst I mangled the code in the area...